node/pkg/watchers/evm/watcher.go

```diff
@@ -202,7 +213,7 @@ func (w *Watcher) Run(parentCtx context.Context) error {
 	defer watcherContextCancelFunc()
 
 	var useFinalizedBlocks bool
-	useFinalizedBlocks, w.safeBlocksSupported, err = w.getFinality(ctx)
+	useFinalizedBlocks, err = w.getFinality(ctx)
```

Comment:
I think not setting `w.safeBlocksSupported` will break ccq.go in that it will always be false even when a chain supports `safe`

Reply:
ln 479 in ccq.go is where it's still used.

Reply:
Confirming that this removal is sane, because the watcher will have safe and finalized block heights populated for all chains.